### DISCUSSION TITLE
Feature request: Add browsersync to bin/setup-grunt

### DISCUSSION THREAD

**Discussion by riccardofinkers** [markshust/docker-magento#607] [Ideas]
Hi Mark, 

I have a feature request that involves adding browser sync with the bin/setup-grunt command so that browser sync works when running the bin/grunt watch command.

**Comment by rangerz** [comment]
How about this
`bin/setup-grunt && bin/grunt watch`

**Reply by riccardofinkers** [reply]
Hi @rangerz,

I'm talking about the last step, because with grunt watch you still need to refresh the page manually and with browsersync it refreshes the page automatically for you on the listening port.
